src/ssh/test-sshtransport.c

```diff
@@ -458,10 +458,6 @@ test_close_problem (TestCase *tc,
 {
   gchar *problem = NULL;
 
-  /* exactly which of the two we get depends on how fast cockpit-ssh shuts down its auth pipe underneath us */
```

Comment:
Should have dropped that comment too. Cleanup in #6122 

Thanks for working this out!